<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReparacionesDialog</class>
 <widget class="QDialog" name="ReparacionesDialog">
  <property name="windowTitle">
   <string>Nueva Reparación</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="labelCliente">
       <property name="text">
        <string>Cliente</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEditCliente"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelMarca">
       <property name="text">
        <string>Marca</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEditMarca"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="labelModelo">
       <property name="text">
        <string>Modelo</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="lineEditModelo"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="labelDescripcion">
       <property name="text">
        <string>Descripción</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QPlainTextEdit" name="plainTextDescripcion"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="labelEstado">
       <property name="text">
        <string>Estado</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QComboBox" name="comboEstado">
       <item>
        <property name="text">
         <string>Pendiente</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Finalizada</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="labelCosto">
       <property name="text">
        <string>Costo</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="inputCosto">
       <property name="placeholderText">
        <string>0.00</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonsLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnGuardar">
       <property name="text">
        <string>Guardar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancelar">
       <property name="text">
        <string>Cancelar</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
